db/migrate/20160219182323_add_bathroom_updates.rb

```diff
@@ -0,0 +1,10 @@
+class AddBathroomUpdates < ActiveRecord::Migration
+  def change
+    create_table :bathroom_updates do |t|
+      t.boolean :occupied
+      t.integer :room
```

Comment:
What I should really do is create a `Bathroom` object with a `name` attribute and a `has_many` relationship to `BathroomUpdates`